src/utils/filterEvents.js

```diff
@@ -0,0 +1,23 @@
+import { format, isAfter } from 'date-fns'
+
+export const filterMeetups = events =>
```

Comment:
probably `getHomepageMeetups` would be a better name? Currently this filters just for the `homepageFeatured` prop and also reverts and transforms the data, so a `get...` name would be more appropriate. The same for the next function

Reply:
MOved and names changed